examples_extra/RepoSearcher/RepoSearcher/IGListCollectionContext+ASDK.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  IGListCollectionContext+ASDK.swift
+//  RepoSearcher
+//
+//  Created by Marvin Nazari on 2017-02-18.
+//  Copyright © 2017 Marvin Nazari. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+import AsyncDisplayKit
+
+extension IGListCollectionContext {
+    func nodeForItem(at index: Int, sectionController: IGListSectionController) -> ASCellNode? {
+        return (cellForItem(at: index, sectionController: sectionController) as? _ASCollectionViewCell)?.node
+    }
```

Comment:
Interesting idea! That cell-based implementation means that we'll get `nil` if the item isn't visible, so we should probably change the implementation to use `nodeForItemAtIndexPath:` under the hood, so that we always get a node.